dsc-website-client/src/pages/CustomizedTimeline.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Timeline from '@material-ui/lab/Timeline';
+import TimelineItem from '@material-ui/lab/TimelineItem';
+import TimelineSeparator from '@material-ui/lab/TimelineSeparator';
+import TimelineConnector from '@material-ui/lab/TimelineConnector';
+import TimelineContent from '@material-ui/lab/TimelineContent';
+import TimelineOppositeContent from '@material-ui/lab/TimelineOppositeContent';
+import TimelineDot from '@material-ui/lab/TimelineDot';
+import FastfoodIcon from '@material-ui/icons/Fastfood';
+import LaptopMacIcon from '@material-ui/icons/LaptopMac';
+import HotelIcon from '@material-ui/icons/Hotel';
+import RepeatIcon from '@material-ui/icons/Repeat';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+import Card from '../components/CustomizedCard'
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    padding: '6px 16px',
+  },
+  secondaryTail: {
+    backgroundColor: theme.palette.secondary.main,
+  },
+}));
+
+export default function CustomizedTimeline() {
+  const classes = useStyles();
+
+  return (
+    <Timeline align="alternate">
```

Comment:
Do you think you could make this into a looping function? As in have all the elements in a list and iterate through them. Would make the code cleaner and have less lines.
For example:
![Screen Shot 2020-10-24 at 2 38 28 PM](https://user-images.githubusercontent.com/43769966/97091010-96c46400-1606-11eb-9c2b-89bd8d6f291c.png)

If this doesn't work or takes too much time, don't worry about it well merge it and come back to it later!

Reply:
yeah sure  I will change it this way.